File: fizzbuzz/fizzbuzz.cpp
#include <iostream>

using namespace std;

int main() {
    for (int i = 1; i <= 100; i++) {
        string ans = "";
        if (i % 3 == 0) ans += "Fizz";
        if (i % 5 == 0) ans += "Buzz";
        if (ans == "") ans = to_string(i);
        cout << ans << endl;
    }
    return 0;
}

File: fizzbuzz/fizzbuzz.hs
-- Functions to get fizzbuzz

fizzbuzz :: [String]
fizzbuzz = map (toFizzBuzz) [1 .. 100]

toFizzBuzz :: Int -> String
toFizzBuzz x
    | (mod x 3 == 0) && (mod x 5 == 0) = "FizzBuzz"
    | mod x 3 == 0 = "Fizz"
    | mod x 5 == 0 = "Buzz"
    | otherwise = show x
